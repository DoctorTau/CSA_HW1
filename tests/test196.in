185
-9439 5680 3578 -5590 -3157 -5717 -6278 9617 8461 -5803 2521 -1150 -2510 4964 6981 7346 7188 -9217 -7924 5251 6796 8221 9300 8987 -8963 -7384 -3119 -8195 -7837 6931 -3042 -704 2332 -7891 8982 7753 -2109 3323 -3686 6992 1822 4561 7773 3297 4167 -7081 5778 5501 7885 -5534 6622 -8928 -7258 8174 709 -8800 9170 4877 4796 -3232 -1181 -5769 -6111 -6083 4257 -6634 8306 2971 9809 7553 3367 -9713 2761 -3728 7186 5701 6120 4673 7081 -9006 -1853 1736 -9189 -3089 4138 -2136 5972 -442 -5286 -7370 8322 2382 3592 -960 5913 1543 -7029 -9277 5688 2785 -3301 4806 7614 -6997 -5659 -4866 -9468 -755 6525 -9563 -8392 9943 220 3963 -2452 -5293 5943 -209 4686 1356 -7290 4198 -2080 -7613 719 8025 -216 -309 2052 -3663 2479 9505 -1633 -8860 -6116 -8122 8605 1791 292 7448 -5129 3252 8713 -9727 937 -7284 4138 2321 6674 273 1317 -9114 -2371 -5237 8563 -8829 6286 1592 -5578 5679 2529 -9288 8820 -2962 -1619 7835 5025 -9461 -4250 -2014 5633 8822 4805 -7385 -7419 5929 -670 829 3520 -7917 6202 7471 -4949 -5120 6031 