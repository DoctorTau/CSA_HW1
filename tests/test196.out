-5590 -6278 -1150 -2510 -7924 -7384 -3042 -704 -3686 -5534 -8928 -7258 -8800 -3232 -6634 -3728 -9006 -2136 -442 -5286 -7370 -960 -4866 -9468 -8392 -2452 -7290 -2080 -216 -8860 -6116 -8122 -7284 -9114 -5578 -9288 -2962 -4250 -2014 -670 -5120 